apps/portals/adknowledgeportal/src/pages/StudyDetailsPage/StudyDetailsPage.tsx

```diff
@@ -42,6 +43,7 @@ const studyDetailsPageTabs: DetailsPageTabConfig[] = [
 const studiesDetailsPageProps: DetailsPageProps = {
   sql: studiesSql,
   sqlOperator: ColumnSingleValueFilterOperator.LIKE,
+  portalDOIConfiguration: studyDoiConfiguration,
```

Comment:
Pass the DOI configuration to the DetailsPage so it can attempt to fetch the DOI